src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/activitiesappointments/ActivitiesAppointmentsClient.kt

```diff
@@ -17,4 +18,17 @@ class ActivitiesAppointmentsClient(private val activitiesAppointmentsApiWebClien
       .bodyToMono(Appointment::class.java)
       .onErrorResume(WebClientResponseException.NotFound::class.java) { Mono.empty() }
       .block()
+
+  fun isAppointmentsRolledOutAt(prisonCode: String) =
+    activitiesAppointmentsApiWebClient
+      .get()
+      .uri("/rollout/{prisonCode}", prisonCode)
+      .retrieve()
+      .bodyToMono(RolloutPrisonPlan::class.java)
+      .onErrorResume(WebClientResponseException.NotFound::class.java) { Mono.empty() }
+      .block()?.appointmentsRolledOut == true
+
+  fun createAppointment() {
+    TODO("to be implemented")
```

Comment:
Along with createAppointment, I think we'll need things like `findAppointment(prison, locationKey, prisonerNumber,  date, startTime, endTime)` so that we can find them to do things like update and cancel, but good for now.

Reply:
Indeed.  It's all very lightweight at the moment.